dev-manuals/test-plans/keycloak/u-keycloak-01-standard-login.md

```diff
@@ -0,0 +1,36 @@
+# Keycloak: standard login (`u.keycloak-01`)
+
+
+Version: V1.0, 25.09.2022 \
+Author: Leon Layer
+
+## Description
+
+Standard login with user email and password.
+
+## Precondition
+
+No user is logged in. If the user was logged in, he has logged out so that he is no longer logged in via the cookies in the browser.
```

Comment:
So far no complaints. The only thing I'm missing is a test that **a new user can register**.